bindings/python/test/test_arrow.py

```diff
@@ -261,6 +263,7 @@ def _create_data(self):
                 "Binary": [b"1", b"23"],
                 "ObjectId": [ObjectId().binary, ObjectId().binary],
                 "Decimal128": [Decimal128(str(i)).bid for i in range(2)],
+                "Code": [str(Code(str(i))) for i in range(2)],
```

Comment:
Can we make this just two strings that are valid code object strings (like for binary) rather than converting back and forth?